ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -393,6 +393,10 @@ aaa_accounting:
     default:
       type: < none | start-stop | stop-only >
       group: < group_name >
+  dot1x:
+    default:
+      type: < none | start-stop | stop-only >
```

Comment:
`none` is not a valid value for this data model. Should be removed here and in the schema.